join_request/niklasp.yml

```
@@ -0,0 +1,70 @@
+collective: Potoc
+member:
+  name: Niklas
+  address: 5CwW67PPdZQQCcdWJVaRJCepSQSrtKUumDAGa7UZbBKwd9R2
+  github: niklasp
```

please fill your matrix handle as per https://github.com/super-collective/collective-cli/pull/4#discussion_r1586380064
```suggestion
  address: 5CwW67PPdZQQCcdWJVaRJCepSQSrtKUumDAGa7UZbBKwd9R2
  matrix:
  github: niklasp
```